.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "8"
+script:
+  - npm run build
+  - npm run bundle
```

Comment:
Shouldn't we do `npm run test` at some point?

Reply:
You are right @okwolf but it is already part of prebuild step https://github.com/spatialillusions/milsymbol/blob/master/package.json#L12

Reply:
@ateshuseyin an that note, isn't `bundle` already part of [`build`](https://github.com/spatialillusions/milsymbol/blob/master/package.json#L13)?

Reply:
Oooww. You are right. `npm run build` is only command we need it. Thanks